monai/data/nifti_saver.py

```diff
@@ -77,29 +92,40 @@ def _create_file_basename(postfix, input_file_name, folder_path, data_root_dir="
         return os.path.join(subfolder_path, filename + "_" + postfix)
 
     def save(self, data, meta_data=None):
-        """Save data into Nifti format file. The metadata should have the following keys:
-            - ``'filename_or_obj'`` -- for output file name creation, corresponding to filename or object
-            - ``'original_affine'`` (optional) for data orientation handling
-            - ``'affine'`` (optional) for data output affine.
-        If meta_data is None, use the detault index from 0 to save data instead.
+        """
+        Save data into a Nifti file.
+        The metadata could optionally have the following keys:
+
+            - ``'filename_or_obj'`` -- for output file name creation, corresponding to filename or object.
+            - ``'original_affine'`` -- for data orientation handling, defaulting to an identity matrix.
+            - ``'affine'`` -- for data output affine, defaulting to an identity matrix.
+            - ``'spatial_shape'`` (optional) for data output shape.
+
+        If meta_data is None, use the default index from 0 to save data instead.
 
         args:
             data (Tensor or ndarray): target data content that save into Nifti format file.
-                Assuming the data shape is `NCHW[D]`.
+                Assuming the data shape starts with spatial dimensions, and followed by time steps or channels.
```

Comment:
Why change the doc-string of data shape?
I think it should be NCHW[D] and seems you kept the logic to swap axis to channel_last:
```py
data = np.moveaxis(data, 0, -1)
```

Reply:
thanks, that's fixed